src/vcpkg/base/messages.cpp

```diff
@@ -490,6 +490,20 @@ namespace
     StdErrMessageSink stderr_sink_instance;
 }
 
+namespace vcpkg
+{
+
+    void MessageSink::println_warning(const LocalizedString& s)
+    {
+        print(Color::warning, msg::format(msg::msgWarningMessage).append(s).append_raw('\n'));
```

Comment:
If this member survives.

```suggestion
        println(Color::warning, msg::format_warning(s));
```